OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -53,7 +53,7 @@ public sealed class DamageType { DamageType() { } }
 
 	public interface IHealthInfo : ITraitInfoInterface
 	{
-		int MaxHP { get; }
+		int DefaultMaxHP { get; }
```

Comment:
```suggestion
		int InitialMaxHitPoints { get; }
```
?

Reply:
Do we really need to rename this at all?

Reply:
'Initial' might be better than 'Default', yeah.

I'm uncomfortable with HitPoints, though.
Not that I disagree with you (though HealthPoints might be more accurate), but if we went for consistency, we'd probably have to replace HP with HitPoints/HealthPoints everywhere in one go (including at yaml level, so with update rules and everything), and I feel that would be a bit excessive and a potential royal pain for a lot of modders and (mod) mappers for no practical benefit.
Basically, if it can be helped at all I'd like to avoid things snowballing out of scope over some minor plumbing change for a downstream feature.

Reply:
Tbh, isn't this better solved by some sort of health modifier trait?

Reply:
Assuming you mean an equivalent to `FirepowerMultiplier`, absolutely _no_, at least not for my usecase.

In my case, an actor can level up from 1 to 100, and its stats are supposed to increase by an exact flat amount each level, rather than a percentage.

Our current set of `*Multiplier` traits and their interfaces are completely unwieldy for that kind of purpose.

Though I guess using some kind of interface instead of modifiying MaxHP directly might still be more sensible.

Reply:
IMO the approach you've taken here is the best solution, although i'm pretty sure that we will need to change `FrozenActor` to track the max as well as current HP to avoid problems with e.g. TD engineer capture vs sabotage. This would then also allow us to remove `IHealthInfo.MaxHP` and any potential confusion about its naming or use.

Reply:
Done.